client/lib/human-date/index.js

```diff
@@ -1,28 +1,24 @@
-import i18n from 'i18n-calypso';
-import moment from 'moment';
-
+import { useTranslate } from 'i18n-calypso';
+import { useMemo, useState } from 'react';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+import { EVERY_TEN_SECONDS, useInterval } from 'calypso/lib/interval';
 const MILLIS_IN_MINUTE = 60 * 1000;
 
-export default function humanDate(
-	dateOrMoment,
-	dateFormat = 'll',
-	locale = i18n.getLocaleSlug()
-) {
-	const now = moment().locale( locale );
-	dateOrMoment = moment( dateOrMoment ).locale( locale );
+function getHumanDateString( now, date, dateFormat, moment, translate ) {
```

Comment:
There's one regression here: the `dateFormat` used to default to `'ll'` but it has no default now. In the devdocs example, that leads to the "5 months ago" example being rendered as `2021-06-11T10:10:42+02:00` instead of the desired `Jun 11, 2021`.

Reply:
good catch 👍🏻 fixed in https://github.com/Automattic/wp-calypso/pull/57826/commits/57a2ae3b1b1a77bf35264cd3d333dd2f2e9d9f24